aviary/interface/default_phase_info/height_energy.py

```diff
@@ -113,17 +106,75 @@ def phase_info_parameterization(phase_info, post_mission_info, aviary_inputs):
     # Altitude
     old_alt_cruise = 32000.0
     if alt_cruise != old_alt_cruise:
-        phase_info['climb']['user_options']['final_altitude'] = (alt_cruise, 'ft')
-        phase_info['cruise']['user_options']['initial_altitude'] = (alt_cruise, 'ft')
-        phase_info['cruise']['user_options']['final_altitude'] = (alt_cruise, 'ft')
-        phase_info['descent']['user_options']['initial_altitude'] = (alt_cruise, 'ft')
+        new_alt = (alt_cruise, 'ft')
+
+        climb = phase_info['climb']
+        user = climb['user_options']
+        if 'altitude_final' in user and user['altitude_final'][0] is not None:
```

Comment:
Is this going to work ok with the current scheme of loading defaults which might overwrite the 'None' flag.

Reply:
It should, since the default actually is None.